lib/icystream.js

```diff
@@ -91,7 +93,7 @@ function getStreamStation(url, callback) {
       return;
     }
     tearDown();
-    console.log(error);
+    console.log("errorCallback in icystream.js : line 107 ->" + error);
```

Comment:
Is this line number correct?  Is there a way to dynamically insert it if this is the best way to surface this error?  Seems like something somebody would forget to update going forward.

Reply:
True. We could add a stack trace. I just put it in there so I knew where my console log statement was coming from.

Reply:
Done, just changed from console.log to console.trace which will provide the current working functions when the error message path is taken.